src/test/java/com/prgms/allen/dining/web/domain/customer/restaurant/CustomerRestaurantApiTest.java

```diff
@@ -64,6 +73,27 @@ void getRestaurants() throws Exception {
 				)));
 	}
 
+	@Test
+	@DisplayName("구매자는 검색한 단어가 포함된 이름을 가진 레스토랑들을 페이징 조회할 수 있다")
+	void getRestaurantsContains() throws Exception {
+
+		final String name = "유명";
+		final int page = 0;
+		final int size = 2;
+
+		mockMvc.perform(get(MessageFormat.format(
+				"/customer/api/restaurants/search?page={0}&size={1}&restaurantName={2}", page, size, name)))
```

Comment:
```suggestion
				MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
		params.add("name", "유명");
		params.add("page", "1");
		params.add("size", "2");

		mockMvc.perform(get("/customer/api/restaurants/search")
				.queryParams(params))
```

**c**: MockHttpServeltRequestBuilder에 이렇게 쿼리 파라미터를 지정하는 메서드가 있어요! (파라미터가 하나인 경우에는 하나의 변수로 지정도 가능해요 ㅎㅎ `.param()` 사용하면 됩니당.)

제가 작성한 방식은 해당하는 Map 형식을 사용하여 파라미터를 만들어 준 것인데 참고 한번 해보시라고 남겨봤어요! 

이런 식으로 작성할 수도 있더라구요 ㅎㅎ

Reply:
이런게 있었군용 ~!! 감사합니다😆

Reply:
[8765a70](https://github.com/prgrms-be-devcourse/BE-03-java-and-dining/pull/27/commits/8765a705d464298e4d9cd9c1d7fbdefbb0ca3add) 반영하였습니다~